ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```diff
@@ -25,13 +25,13 @@
 
 - name: Add AdoptOpenJDK GPG key
   apt_key:
-    url: https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+    url: https://packages.adoptium.net/artifactory/api/gpg/key/public
```

Comment:
This should be a separate PR

Reply:
This has been reverted, and a seperate PR is on its way.